server/org_config.go

```diff
@@ -0,0 +1,187 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/influxdata/chronograf"
+)
+
+type organizationConfigResponse struct {
+	Links selfLinks `json:"links"`
+	chronograf.OrganizationConfig
+}
+
+func newOrganizationConfigResponse(config chronograf.OrganizationConfig) *organizationConfigResponse {
+	return &organizationConfigResponse{
+		Links: selfLinks{
+			Self: "/chronograf/v1/org_config",
+		},
+		OrganizationConfig: config,
+	}
+}
+
+type logViewerConfigResponse struct {
+	Links selfLinks `json:"links"`
+	chronograf.LogViewerConfig
+}
+
+func newLogViewerConfigResponse(config chronograf.OrganizationConfig) *logViewerConfigResponse {
```

Comment:
Why not just have this be a function of `chronograf.LogViewerConfig`?

Reply:
This is how `newAuthConfigResponse` was handled in server/config.go, so we followed that pattern

Reply:
I think I'd do something like

```go
func newLogViewerConfigResponse(config chronograf.LogViewerConfig) 
```

Reply:
👍